_commands/basics/man.md

```diff
@@ -3,22 +3,31 @@
 
 man
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`man` provides *manual pages* for terminal commands
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ man git
+q // to quit
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 

```

Comment:
Add an example of how to lookup the introductory text for each section of the man pages and a short explanation of the structure of the different section of man pages could be useful.